orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/cuda/adam_optimizer/compat.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// This file is adapted from microsoft/DeepSpeed
+// compat.h
+
+/* Copyright 2020 The Microsoft DeepSpeed Team
+   Copyright NVIDIA/apex
+   This file is adapted from fused adam in NVIDIA/apex, commit a109f85
+*/
+
+#ifndef TORCH_CHECK
+#define TORCH_CHECK AT_CHECK
+#endif
+
+#ifdef VERSION_GE_1_3
```

Comment:
This macro won't work, since we didn't have this macro defined anywhere. 

However, I don't think we would need this macro either, as we will only support torch>1.8. 

Just replace the use of "DATA_PTR" with  "data"

Reply:
remove this head file, and all the #include "compat.h" occurrences.